pkg/apis/core/types_shoot.go

```diff
@@ -351,17 +351,19 @@ type DNS struct {
 	Domain *string
 	// Providers is a list of DNS providers that shall be enabled for this shoot cluster. Only relevant if
 	// not a default domain is used.
+	// Deprecated: Configuring multiple DNS providers is deprecated and will be forbidden in a future release. Please use the DNS extension provider config for additional providers.
```

Comment:
Better use the full name of the shoot-dns-service extension.
```suggestion
	// Deprecated: Configuring multiple DNS providers is deprecated and will be forbidden in a future release. Please use the `shoot-dns-service` extension provider config for additional providers.
```

Reply:
How can we know that it's the `shoot-dns-service`?

Reply:
I understand the intention to be neutral. As there is no known other extension listed on [Known extension implementations list](https://github.com/gardener/gardener/blob/b26920f63445597d107fccadd05a19cc5783ca2c/extensions/README.md#known-extension-implementations), I don't see an advantage not to named it.
But it was only a suggestion.

Reply:
Okay, thanks. Pease see the adjusted doc string.